From 46a1580018af71435dc60945f1ee5b1ab00fca92 Mon Sep 17 00:00:00 2001
From: Alfred Wingate <parona@protonmail.com>
Date: Sat, 23 Dec 2023 21:50:14 +0200
Subject: [PATCH] libgimpbase libgimpwidgets: install needed headers

* These headers weren't installed despite being included by other
  installed headers.

Bug: https://gitlab.gnome.org/GNOME/gimp/-/issues/10132
Bug: https://bugs.gentoo.org/920535
Signed-off-by: Alfred Wingate <parona@protonmail.com>
---
 libgimpwidgets/meson.build | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libgimpwidgets/meson.build b/libgimpwidgets/meson.build
index 91c11388a2e..cadd3bcfa5c 100644
--- a/libgimpwidgets/meson.build
+++ b/libgimpwidgets/meson.build
@@ -145,6 +145,7 @@ libgimpwidgets_headers_introspectable = files(
   'gimplabelintwidget.h',
   'gimplabelspin.h',
   'gimplabelentry.h',
+  'gimplabelstringwidget.h',
   'gimpintstore.h',
   'gimpmemsizeentry.h',
   'gimpnumberpairentry.h',
-- 
GitLab

